src/sphinx_autodoc_typehints/__init__.py

```diff
@@ -194,6 +194,9 @@ def format_annotation(annotation: Any, config: Config) -> str:  # noqa: C901, PL
     if isinstance(annotation, tuple):
         return format_internal_tuple(annotation, config)
 
+    if isinstance(annotation, TypeAliasForwardRef):
+        return str(annotation)
```

Comment:
Should we check whether we can make an xref here? Perhaps in a followup.

Reply:
sorry, I'm not following. Could you elaborate?

Reply:
wait I messed up the review process. I didn't mean to dismiss your review. my bad